api/tasks.py

```diff
@@ -4,17 +4,19 @@
 from wordpress.user import WPUser
 from lazona_connector.celery import app
 
-
 @app.task
 def create_delivery(order):
-    delivery = Client(order).create_delivery()
-
-    if type(delivery).__name__ == 'Error':
-        return delivery.to_dict()
-    else:
-        delivery.print_pdf()
-        return {'success': 'all good'}
-
+    deliveries_by_vendor = Client(order).create_delivery()
+    deliveries_success = []
+    deliveries_error = []
+    for delivery in deliveries_by_vendor:
+        if type(delivery).__name__ == 'Error':
```

Comment:
we should probably implement a `status` method on `Delivery` and `Error` which would be slightly better. We wouldn't couple to the exact class type.

Reply:
I've refactored so that ._is_errored is now a Delivery class' method and all deliveries returned by Koiki's are of that class type. Not sure if we will need the Error class in the future.